concepts/functions/about.md

```diff
@@ -1,3 +1,176 @@
 # About
 
-TODO
+Functions are bodies of code containing one or more [statements or expressions][statements or expressions]. A function may optionally
+[return an expression][return value]. (Recall that a statement is an instruction that does not return a value. An expression evaluates to
+a resulting value.)
```

Comment:
```suggestion
[return an value][return value]. 
```

- Macros can return expressions, but functions can only return values.
- People who don't remember the distinction between statements and expressions can just click the link.